pkg/kernel/filter/defaults.go

```diff
@@ -37,7 +37,7 @@ var defaultSyscalls = []string{
 	"sys_mkdirat",
 	"sys_statfs",
 	"sys_access",
-	"sys_prlimit",
+	"sys_prlimit64",
```

Comment:
There is no such thing as `sys_prlimit` - no idea what I was thinking here originally.